.github/actions/prepare/action.yml

```diff
@@ -23,3 +23,4 @@ runs:
       run: |
         python manage.py makemigrations
         python manage.py migrate
+        python manage.py loaddata opportunityboard/fixtures/category_data.json
```

Comment:
It's not clear to me why we need to inject this data during CI. Tests should be self contained and therefore they should add any data they rely on being in the database at test execution time.

Reply:
This is not exactly testing. This is let Django load some important hard-coded categories into the database so users can actually see them in the UI.

Reply:
Right and the database being used during the GitHub actions workflows is _only_ for testing, thus these are not required.